src/kumo-api.ts

```diff
@@ -197,7 +197,8 @@ export class KumoApi {
 
         if (existingDeviceIndex !== undefined) {
           this.log.info('Updated existing device. Serial: %s. Label: %s', device.serial, device.label);
-          this.devices[existingDeviceIndex] = device;
+          this.devices[existingDeviceIndex].label = device.label;
+          this.devices[existingDeviceIndex].zoneTable = device.zoneTable;
```

Comment:
Only update the label and zoneTable. Preserve the overrideAddress and any other info that we add later